### ISSUE TITLE
Merge rights

### ISSUE THREAD

**Issue by marqh** [cf-convention/cf-conventions#63]
I think there is a need to agree which users have merge rights to this repository and implement the permissions.

I provided review and merge services on this repository's predecessor, but I am unable to provide this service now, as my permissions do not allow it.

Please may we create a team to manage merging of content onto this repository and agree its membership?

thank you
mark

**Comment by rhattersley** [comment]
:+1: ... for example, see https://github.com/cf-convention/cf-conventions/pull/62#issuecomment-169985542

**Comment by mattben** [comment]
Agreed @marqh 

I've updated the teams page (github teams changed a while back). I have created two teams `web` for the  site and `conventions` for all the other repos.

@rsignell-usgs and @painter1 as I am solely a web dude, can you two please populate these teams with the appropriate people.

**Comment by painter1** [comment]
We have 16 people in the Contributors team.  It has write access, so all should be able to approve pull requests.   @mattben, how should that team differ from your new "conventions" team?  Anyone, do you have opinions on who belongs in the "conventions" team?  Everyone in Contributors?

**Comment by japamment** [comment]
Can someone please explain to me what is happening with the various cf-conventions repositories on github? I have just tried to push something to cf-convention/cf-convention.github.io.git (to correct an error in the standard name table) and received a permission denied message. This worked fine about a month ago. Do I need to be added back in as a Member?

**Comment by mattben** [comment]
@japamment yes, we are waiting on @rsignell-usgs and @painter1 to assign people to the different development teams. in the mean time you can create a pull request from another branch or your fork and I can merge it. (but only for the website repo)

**Comment by japamment** [comment]
OK, thanks, I have created a pull request from japamment/cf-convention.github.io to cf-convention/cf-convention.github.io – would you be able to merge it, please?

**Comment by rhattersley** [comment]
> we are waiting on @rsignell-usgs and @painter1 to assign people to the different development teams

1. Please can someone (@mattben? @rsignell-usgs?) give the "conventions" team write access to this repo?
2. Please can we (@rsignell-usgs? @painter1?) expand the membership of the "conventions" team so we can start merging some pull requests and making progress on 1.7?

With (1) done I'd (reluctantly) get @cf-conventions-bot to merge its own pull-request, #71, as that's just an infrastructure change, but I would *not* want it to be used to merge content changes.

**Comment by dblodgett-usgs** [comment]
It seems this issue has been resolved? If not, maybe it should be closed and reopened with a clear purpose?

**Comment by dblodgett-usgs** [comment]
This issue will be further discussed in and is superseded by #172 and #173.
